ScanbotSDKExamples/NET/ClassicComponent.iOS/ViewControllers/MainViewController.cs

```diff
@@ -0,0 +1,438 @@
+// This file has been autogenerated from a class added in the UI designer.
+
+using System.Diagnostics;
+using ClassicComponent.iOS.Models;
+using ClassicComponent.iOS.Utils;
+using ClassicComponent.iOS.ViewControllers;
+using DocumentSDK.MAUI.Native.iOS.Configurations;
+using Scanbot.ImagePicker.iOS;
+using ScanbotSDK.iOS;
+using ScanbotSDK.MAUI.Constants;
+using ScanbotSDK.MAUI.Models;
+using SBSDK = ScanbotSDK.MAUI.Native.iOS.ScanbotSDK;
+
+
+namespace ClassicComponent.iOS
+{
+    internal interface ISDKServiceSource
+    {
+        List<SDKService> SDKServices { get; set; }
+        void RowSelected(int index);
+    }
+
+    interface IModifyDocumentContrllerDelegate
+    {
+        void DidUpdateDocumentImage(UIImage updatedImage);
+    }
+
+    internal interface ICameraDemoViewControllerDelegate
+    {
+        void DidCaptureDocumentImage(UIImage documentImage, UIImage originalImage);
+    }
+
+    public partial class MainViewController : UIViewController, ISDKServiceSource, ICameraDemoViewControllerDelegate, IModifyDocumentContrllerDelegate
+    {
+        public List<SDKService> SDKServices { get; set; }
+        public ProgressHUD ProgressIndicator { get; private set; }
+
+        private bool _isBusy;
+        public bool IsBusy
+        {
+            get { return _isBusy; }
+            set
+            {
+                _isBusy = value;
+                ProgressIndicator?.ToggleLoading(value);
+            }
+        }
+
+        private UIImage originalDocumentImage, editedDocumentImage;
+
+        public MainViewController(IntPtr handle) : base(handle) { }
+
+        public override void ViewDidLoad()
+        {
+            base.ViewDidLoad();
+            this.NavigationItem.Title = Texts.AppTitle;
+            ProgressIndicator = ProgressHUD.Load(View.Frame);
+            LoadFeatures();
+            SetUpTableView();
+            UpdateHintLabel(string.Empty);
+            UpdateDocumentImageView(null);
+            SetTapGestureToDocumentImage();
+        }
+
+        private void LoadFeatures()
+        {
+            SDKServices = new List<SDKService>
+            {
+                new SDKService { Title = SDKServiceTitle.ScanningUI },
```

Comment:
@scan-mayank, let's rather keep the code here similar to the new changes made for the other example apps, by adding a method reference to the list and then invoking the code that way.

Perhaps we may also call it a SDKFeature as service isn't a good reflection of the functionality and is again consistent with changes made in other samples. 

Finally, I think we can have the string titles here directly since this is the only file that uses those constants so we may use the string literals here directly.